packages/remix/test/fixtures/13-remix-v2-public-path/remix.config.js

```diff
@@ -0,0 +1,8 @@
+/** @type {import('@remix-run/dev').AppConfig} */
+export default {
+  ignoredRouteFiles: ["**/.*"],
+  // appDirectory: "app",
+  assetsBuildDirectory: "public",
+  publicPath: "../..",
```

Comment:
In the original PR, the publicPath value was reported as `/apps/my-app/build/`, which the current code ends up resolving to `../..`. I don't think the intent was that `../../` was the value explicitly.

Reply:
No prob, I'll fix.